package.json

```diff
@@ -24,7 +24,7 @@
   },
   "scripts": {
     "build:altis": "npm run prepublishOnly && npm run copynota",
-    "copynota": "cp -a \"./dist/.\" \"../../web-dev/packages/nota/node_modules/@altis/cornerstone-tools/dist\"",
+    "copynota": "cp -a \"./dist/.\" \"../altis-labs/nota/packages/nota/node_modules/@altis/cornerstone-tools/dist\"",
```

Comment:
Normally I would say revert this, but since it was already hardcoded in the first place, no need.